core/includes/common.inc

```diff
@@ -7132,81 +7149,39 @@ function backdrop_sort(array &$array, array $keys = array('weight'), $dir = SORT
   }
   $keys = $new_keys;
 
-  // If sorting on a single value, optimize the sorting to sort only on that
```

Comment:
I'm curious if there's a way to simplify for case where sorting by a single value, like before.

Reply:
I'm not sure. You can read about the PHP sorting function we're now using here: https://www.php.net/manual/en/function.array-multisort But I couldn't find any references there to optimisation, performance, etc...